utest.h

```diff
@@ -128,11 +128,11 @@ typedef uint64_t utest_uint64_t;
 #endif
 
 #pragma section(".CRT$XCU", read)
-#define UTEST_INITIALIZER(f)                                                   \
-  static void __cdecl f(void);                                                 \
-  __pragma(comment(linker, "/include:" UTEST_SYMBOL_PREFIX #f "_"));           \
-  UTEST_C_FUNC __declspec(allocate(".CRT$XCU")) void(__cdecl * f##_)(void) =   \
-      f;                                                                       \
+#define UTEST_INITIALIZER(f)                                                 \
```

Comment:
It looks like some of this has been formatted with clang-format, but with a different configuration to the default or something? Any ideas?

Reply:
I have no clue basically? I am using VS Code ... on W10. How do I switch it off?

Reply:
Nice! Thanks for this 😄